etcd/datadog_checks/etcd/etcd.py

```diff
@@ -64,12 +62,27 @@ class Etcd(OpenMetricsBaseCheck):
     }
 
     def __init__(self, name, init_config, agentConfig, instances=None):
+
         if instances is not None:
             for instance in instances:
                 # For legacy check ensure prometheus_url is set so
                 # OpenMetricsBaseCheck instantiation succeeds
                 if not is_affirmative(instance.get('use_preview', False)):
                     instance.setdefault('prometheus_url', '')
```

Comment:
should prometheus_url be remapped to url?

Reply:
The wrapper doesn't have a `url` option.`url`s are often modified in the check logic by additional paths, I'm not sure if there are benefits to mapping it in the wrapper.

Reply:
Oh, sorry I was asking because I did not see prometheus url in the config, but it was in the not shown part of the diff. My bad